### ISSUE TITLE
Tree "row1 selected" read out when item gets collapsed

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52177]
Environment Details:
VSCode Version : 1.24.0
Mac Version: HighSierra10.13.5 

Additional Details:
MAS Violated: MAS1.3.1

Repro Steps:
1. Launch VS Code while VoiceOver is ON.
2. Go to View menu and Select Problems.
3. Collapse the json file under Problems tab and check what voiceover is reading.

Actual:
VoiceOver reads it as "row1 selected" when we collapse the json file. It does not read the name of the file and its path and how many problems detected under it.

Expected:
Voiceover should read the file name with its state "collapsed" and its path with how many problems identified.
 
Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not get the proper information about the controls.

MAS Reference:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for reference: 
[A11y VSCode Problems VoiceOver collapsed state.pptx](https://github.com/Microsoft/vscode/files/2110882/A11y.VSCode.Problems.VoiceOver.collapsed.state.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by sandy081** [comment]
@AccessibilityTestingTeam-TCS It reads out the file name and number of problems for me.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Open more than two files which has errors then try reading it. Because I faced the same issue several times.

**Comment by isidorn** [comment]
I can repro this in VoiceOver, collapsing a file in problems view reads out "row1 collapsed". A more informative message would be useful.

**Comment by isidorn** [comment]
This is actually a tree issue, that once an item is collapsed and a user navigates over it sometiems the tree simply announces "row1 collapsed", moving to @joaomoreno

**Comment by joaomoreno** [comment]
Closing it as designed. The name was already read out once the user focused it. No need to repeat it. NVDA simply reads `collapsed` and `expanded`, btw.
